README.md

```diff
@@ -86,6 +86,7 @@ db.SetMaxIdleConns(10)
 `db.SetMaxOpenConns()` is highly recommended to limit the number of connection used by the application. There is no recommended limit number because it depends on application and MySQL server.
 
 `db.SetMaxIdleConns()` is recommended to be set same to (or greater than) `db.SetMaxOpenConns()`. When it is smaller than `SetMaxOpenConns()`, connections can be opened and closed very frequently than you expect. Idle connections can be closed by the `db.SetConnMaxLifetime()`. If you want to close idle connections more rapidly, you can use `db.SetConnMaxIdleTime()` since Go 1.15.
+It's confuse here! The code shows if MaxIdleConns is large than MaxOpen, will be reduce to MaxOpen
```

Comment:
Does this line need?

It seems that removing "(or greater than)" is enough to resolve this issue.
https://github.com/go-sql-driver/mysql/pull/1141#issuecomment-846369169

Reply:
Yes，I will fix as that. This line is just used to open a MR

Reply:
Done! @shogo82148